app/components/collections/release_component.rb

```diff
@@ -12,6 +12,18 @@ def initialize(collection:)
     sig { returns(Collection) }
     attr_reader :collection
 
-    delegate :release_option, :release_duration, :access, to: :collection
```

Comment:
Can you say more about this change?

Reply:
We're no longer using `release_option` or `release_duration` in the component template, replaced with new `release_info`. Happy to restore if it is used elsewhere.

Reply:
OK, that makes sense then not to expose them in the public interface. Good call, thanks!